cpu/native/netdev_tap/netdev_tap.c

```diff
@@ -222,9 +222,9 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
             }
             */
 
-            static uint8_t buf[ETHERNET_FRAME_LEN];
+            static uint8_t staticbuf[ETHERNET_FRAME_LEN];
```

Comment:
Maybe `nullbuf` to reflect the fact that this buffer is discarded immediately?

Reply:
amended